packages/api-v2/src/APIProvider.tsx

```diff
@@ -68,148 +42,149 @@ const getWebSocketURL = () => {
     const endpoint = getSocketURL();
     const app_id = getAppId();
     const language = localStorage.getItem('i18n_language');
-    const brand = 'deriv';
-    const wss_url = `wss://${endpoint}/websockets/v3?app_id=${app_id}&l=${language}&brand=${brand}`;
-
-    return wss_url;
+    return `wss://${endpoint}/websockets/v3?app_id=${app_id}&l=${language}&brand=deriv`;
 };
 
-/**
- * Retrieves or initializes a WebSocket instance based on the provided URL.
- * @param {string} wss_url - The WebSocket URL.
- * @returns {WebSocket} The WebSocket instance associated with the provided URL.
- */
-const getWebsocketInstance = (wss_url: string, onWSClose: () => void, onOpen?: () => void) => {
-    if (!window.WSConnections) {
-        window.WSConnections = {};
-    }
-
-    const existingWebsocketInstance = window.WSConnections[wss_url];
-    if (
-        !existingWebsocketInstance ||
-        !(existingWebsocketInstance instanceof WebSocket) ||
-        [2, 3].includes(existingWebsocketInstance.readyState)
-    ) {
-        window.WSConnections[wss_url] = new WebSocket(wss_url);
-        window.WSConnections[wss_url].addEventListener('close', () => {
-            if (typeof onWSClose === 'function') onWSClose();
-        });
-
-        window.WSConnections[wss_url].addEventListener('open', () => {
-            if (typeof onOpen === 'function') onOpen();
-        });
-    }
-
-    return window.WSConnections[wss_url];
-};
-
-/**
- * Retrieves the active WebSocket instance.
- * @returns {WebSocket} The WebSocket instance associated with the provided URL.
- */
-export const getActiveWebsocket = () => {
-    const wss_url = getWebSocketURL();
-
-    return window?.WSConnections?.[wss_url];
-};
+const APIContext = createContext<APIContextData | null>(null);
 
 /**
- * Initializes a DerivAPI instance for the global window. This enables a standalone connection
+ * Initializes a derivAPIRef instance for the global window. This enables a standalone connection
  * without causing race conditions with deriv-app core stores.
- * @returns {DerivAPIBasic} The initialized DerivAPI instance.
+ * @returns {derivAPIRefBasic} The initialized derivAPIRef instance.
  */
 const initializeDerivAPI = (onWSClose: () => void, onOpen?: () => void): DerivAPIBasic => {
-    if (!window.DerivAPI) {
-        window.DerivAPI = {};
-    }
-
     const wss_url = getWebSocketURL();
-    const websocketConnection = getWebsocketInstance(wss_url, onWSClose, onOpen);
 
-    if (!window.DerivAPI?.[wss_url] || window.DerivAPI?.[wss_url].isConnectionClosed()) {
-        window.DerivAPI[wss_url] = new DerivAPIBasic({ connection: websocketConnection });
-    }
+    const connection = new WebSocket(wss_url);
+    connection.addEventListener('close', () => {
+        if (typeof onWSClose === 'function') onWSClose();
+    });
 
-    return window.DerivAPI?.[wss_url];
-};
+    connection.addEventListener('open', () => {
+        if (typeof onOpen === 'function') onOpen();
+    });
```

Comment:
```suggestion
    connection.addEventListener('open', () => onOpen?.());
```
could we refactor it to optional chaining? 🤔